money.gemspec

```diff
@@ -1,69 +1,29 @@
-# Generated by jeweler
-# DO NOT EDIT THIS FILE DIRECTLY
-# Instead, edit Jeweler::Tasks in Rakefile, and run 'rake gemspec'
 # -*- encoding: utf-8 -*-
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
```

Comment:
Why do we need this? http://guides.rubygems.org/patterns/#mangling-the-load-path

Reply:
👍  will remove